packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div ref="range" type="range" class="sf-range" :disabled="disabled">
+    <slot v-bind="$attrs" />
+  </div>
+</template>
+<script>
+import noUiSlider from "nouislider";
+export default {
+  name: "SfRange",
+  props: {
+    /*
+     * Sets the number of sliders (1 or 2) by adding initial values to the array
+     */
+    value: {
+      type: [Number, Array],
+      default: () => [0, 1],
+    },
+    /*
+     * Disabling the slider
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Settings for noUiSlider library
+     */
+    config: {
+      type: Object,
+      default: () => {
+        return {
+          start: [0, 1],
+          range: {
+            min: 0,
+            max: 10,
+          },
+          step: 1,
+          connect: true,
+          direction: "ltr",
+          orientation: "horizontal",
+          behaviour: "tap-drag",
+          tooltips: false,
+          keyboardSupport: true,
+        };
+      },
+    },
+  },
+  watch: {
+    config: {
+      handler(newConfig) {
+        return this.resetAndChangeOption(newConfig);
+      },
+      deep: true,
+    },
+    value: {
+      handler(values) {
+        return this.$refs.range.noUiSlider.set(values);
+      },
+      immedite: true,
```

Comment:
I don't think this will work ;) should be `immediate` https://vuejs.org/v2/api/#watch

Reply:
thanks :)